Add homepage

guilhermesimoes opened the pull request:
This way it is possible to access the gem's source code from the gem's page on rubygems.org
https://rubygems.org/gems/nullobject

nilbus reviewed (approved):
This seems like the most sensible homepage to me.

guilhermesimoes committed:
Add homepage to gemspec

This way it is possible to access the gem's source code from the gem's
page on rubygems.org
https://rubygems.org/gems/nullobject

martinciu committed:
Merge branch 'master' into patch-1

martinciu commented:
Thanks for contribution
